nav2_behavior_tree/include/nav2_behavior_tree/plugins/action/spin_action.hpp

```diff
@@ -65,7 +65,9 @@ class SpinAction : public BtActionNode<nav2_msgs::action::Spin>
         BT::InputPort<double>("time_allowance", 10.0, "Allowed time for spinning"),
         BT::InputPort<bool>("is_recovery", true, "True if recovery"),
         BT::OutputPort<ActionResult::_error_code_type>(
-          "error_code_id", "The spin behavior error code")
+          "error_code_id", "The spin behavior error code"),
+        BT::OutputPort<ActionResult::_error_code_type>(
+          "error_msg", "The spin behavior error msg"),
```

Comment:
Here and for all other BT headers: this is a string type, not a `_error_code_type` (ie int)

Reply:
Done.